src/installation/install_epinio_on_rancher.md

```diff
@@ -0,0 +1,56 @@
+# Rancher configuration
+
+Before installing Epinio, you need a running Rancher instance.
+
+If you don't get it installed yet, feel free to follow installation guides [here](https://rancher.com/docs).
+
+## Epinio Prerequisites
+
+### Kubernetes cluster
+The supported configuration is to either deploy a new cluster or import an existing one in Rancher (those steps are detailed in the Rancher docs).</br>
```

Comment:
```suggestion
The two supported methods are to either deploy a new cluster or to import an existing one into Rancher (those steps are detailed in the Rancher docs).</br>
```

Is the reference to detailed steps for deploying, for importing, or both ? (The phrasing above is ambiguous to me. Recommend splitting that part of into its own sentence).

Reply:
Deploying or importing are explained [here](https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/).
I can mention it but it will stick to v2.6.

Reply:
Ok. So the reference is for both methods. I change my proposal to:
```suggestion
The two supported methods are to either deploy a new cluster or to import an existing one into Rancher. The steps for take for either method are detailed in the Rancher documentation (See `Cluster Provisioning`).</br>
```